packages/driver/cypress/e2e/dom/visibility.cy.ts

```diff
@@ -723,23 +723,18 @@ describe('src/cypress/dom/visibility', () => {
     describe('width and height', () => {
       it('is hidden if offsetWidth is 0', function () {
         expect(this.$divNoWidth.is(':hidden')).to.be.true
-        expect(this.$divNoWidth.is(':visible')).to.be.false
-
         expect(this.$divNoWidth).to.be.hidden
-        expect(this.$divNoWidth).to.not.be.visible
-
         cy.wrap(this.$divNoWidth).should('be.hidden')
-        cy.wrap(this.$divNoWidth).should('not.be.visible')
```

Comment:
Can you explain why these tests are removed?

Reply:
Name of test was is hidden and test should not check visibility in this scenario. Other test that check it's visibility could be added. Still visibility and is hidden is not the same and for this test it was not essential to determine if this element was visible or not.

Reply:
@senpl Please break these same checks out into another test for visibility. We want to keep the tests having the same results and checking the same things to avoid breaking changes for users.

Reply:
added.